sdks/sdk-core/src/addresses.ts

```diff
@@ -271,6 +272,16 @@ const ZERO_ADDRESSES: ChainAddresses = {
   swapRouter02Address: '0xD936711eABD2Ce52747d7122757316C7DFe3599b',
 }
 
+const BOB_ADDRESSES: ChainAddresses = {
+  v3CoreFactoryAddress: '0xcb2436774C3e191c85056d248EF4260ce5f27A9D',
+  multicallAddress: '0x5d6b0f5335ec95cD2aB7E52f2A0750dd86502435',
+  quoterAddress: '0x6Aa54a43d7eEF5b239a18eed3Af4877f46522BCA',
+  // v3MigratorAddress: 'NA',
```

Comment:
Hey @msvstj, This contract was not filled in the spreadsheet so I comment it out.